app/javascript/react/components/Navbar/Navbar.tsx

```diff
@@ -35,95 +37,116 @@ const Navbar = () => {
 
   return (
     <S.Header>
-      <a
-        href={urls.habitatMap}
-        aria-label={t("navbar.sections.aircastingPage")}
-      >
-        <S.AircastingLogo alt={t("navbar.altLogo")} src={logo} />
-      </a>
-      <S.Container>
-        <S.BuyCTA href={urls.airbeamBuyNow}>
-          {t("navbar.sections.getAirbeam")}
-        </S.BuyCTA>
-        <nav>
-          <S.MenuButton onClick={() => setNavMenuVisible(true)}>
-            <img
-              src={hamburger}
-              alt={t("navbar.altMenu")}
-              aria-label={t("navbar.sections.openMenu")}
-            ></img>
-          </S.MenuButton>
-        </nav>
-      </S.Container>
-      <S.NavList $isVisible={navMenuVisible}>
-        <S.NavHeader>
-          <a
-            href={urls.habitatMap}
-            aria-label={t("navbar.sections.habitatMapPage")}
-          >
-            <img src={habitatMapLogo} alt={t("navbar.altHabitatMapLogo")} />
-          </a>
-          <S.Button onClick={() => setNavMenuVisible(false)}>
+      <S.MobileContainer>
+        <S.GoBack href={urls.map} aria-label={t("navbar.mapPage")}>
+          <img
+            src={goBackIcon}
+            alt={t("navbar.altGoBackIcon")}
+            aria-label={t("navbar.goBack")}
+          />
+          {t("navbar.goBack")}
+        </S.GoBack>
+      </S.MobileContainer>
+      <S.DesktopContainer>
+        <a
+          href={urls.habitatMap}
+          aria-label={t("navbar.sections.aircastingPage")}
+        >
+          <S.AircastingLogo alt={t("navbar.altLogo")} src={logo} />
+        </a>
+        <S.Container>
+          <S.BuyCTA href={urls.airbeamBuyNow}>
+            {t("navbar.sections.getAirbeam")}
```

Comment:
This button has blue -ish text tight now and turns gray when visited. Is this intentional? On the design, I can see it is all the time gray400